hhs_oauth_server/middleware.py

```diff
@@ -0,0 +1,11 @@
+from django.utils.deprecation import MiddlewareMixin
+from rest_framework.exceptions import APIException
+from django.http import HttpResponse
+
+
+# Mixin for compatibility with Django <1.10
```

Comment:
We're on Django 1.11.9.

Reply:
yeah but we're using `MIDDLEWARE_CLASSES` which is the pre 1.10 version https://docs.djangoproject.com/en/2.0/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware

Reply:
Okay, that's fascinating. It would be good to upgrade at some point, but maybe that's outside of scope here. I'll add a ticket to the backlog.

Reply:
Okay, it's CBBP-904